plotter/example_heat_rasterized_test.go

```diff
@@ -0,0 +1,51 @@
+// Copyright ©2020 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package plotter_test
+
+import (
+	"log"
+	"os"
+
+	"gonum.org/v1/gonum/mat"
+	"gonum.org/v1/plot"
+	"gonum.org/v1/plot/palette"
+	"gonum.org/v1/plot/plotter"
+	"gonum.org/v1/plot/vg/draw"
+	"gonum.org/v1/plot/vg/vgimg"
+)
+
+func ExampleHeatMap_rasterized() {
+	m := offsetUnitGrid{
+		XOffset: -2,
+		YOffset: -1,
+		Data: mat.NewDense(3, 4, []float64{
+			1, 2, 3, 4,
+			5, 6, 7, 8,
+			9, 10, 11, 12,
+		})}
+
+	pal := palette.Heat(12, 1)
+	plt, err := plot.New()
+	if err != nil {
+		log.Panic(err)
+	}
+
+	img := vgimg.New(250, 175)
+	dc := draw.New(img)
+
+	raster := plotter.NewHeatMap(&m, pal)
+	raster.Rasterized = true
+	plt.Add(raster)
+	plt.Draw(dc)
+	w, err := os.Create("testdata/rasterHeatMap.png")
+
+	if err != nil {
+		log.Panic(err)
+	}
+	png := vgimg.PngCanvas{Canvas: img}
+	if _, err = png.WriteTo(w); err != nil {
+		log.Panic(err)
+	}
```

Comment:
Perhaps simplify a bit the example to just use the `plot.Plot.Save` method?

```suggestion
	raster := plotter.NewHeatMap(&m, pal)
	raster.Rasterized = true
	plt.Add(raster)

	err := plt.Save(5*vg.Centimeter, 3.5*vg.Centimeter, "testdata/rasterHeatMap.png")
	if err != nil {
		log.Panic(err)
	}
```